src/test/java/picard/sam/markduplicates/CheckDuplicateMarkingTest.java

```diff
@@ -0,0 +1,35 @@
+package picard.sam.markduplicates;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class CheckDuplicateMarkingTest {
+    private static final String TEST_FILES_DIR = "testdata/picard/sam/CheckDuplicateMarking";
+
+    @DataProvider(name = "checkDuplicateMarkingDataProvider")
+    public Object[][] checkDuplicateMarkingDataProvider() {
+        return new Object[][]{
+                {Paths.get(TEST_FILES_DIR, "pass_queryname.sam"), 0},
```

Comment:
it's a minor thing but you can avoid the duplication here by changing this to
```suggestion
                {"pass_queryname.sam", 0},
```
And use `Paths.get(TEST_FILES_DIR, input)` below.